packages/@okta/vuepress-site/docs/guides/sign-into-web-app/define-callback/springboot/define-route.md

```diff
@@ -1,6 +1,6 @@
 The Okta Spring Boot Starter configures and hosts this route for you. By default, this route is hosted at `/login/oauth2/code/okta`.
 
-To following along with the defaults on the next page set the property `okta.oauth2.redirect-uri` to `/authorization-code/callback` in your `src/main/resources/application.properties` file:
+To following along with this guide, set the property `okta.oauth2.redirect-uri` to `/authorization-code/callback` in your `src/main/resources/application.properties` file:
```

Comment:
Should that be "follow" (without "ing")?

Reply:
fixed, thanks @motikagan-okta